.github/workflows/ccv_bootcamp-summer-tag.yml

```diff
@@ -4,17 +4,17 @@ on: [workflow_dispatch]
 env:
   CLASS: ccv_bootcamp
   TARGET: r_julia
-  SQLITE: true
-  PYTHON_VERSION: 3.7
+  SQLITE: false
+  PYTHON_VERSION: "3.9.17"
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-        #https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/
+      - uses: actions/checkout@v3.5.3
```

Comment:
The latest checkout version is 4. Any reason to keep it locked to this version? 

https://github.com/actions/checkout